shared/src/components/icons.tsx

```diff
@@ -26,6 +26,9 @@ import {
   Launch,
   MoreVert,
 } from '@material-ui/icons';
+import BlockIcon from '@material-ui/icons/Block';
```

Comment:
Don't forget to bump the version for the shared package since this introduces new changes.